realm/realm-library/src/main/java/io/realm/internal/ColumnIndices.java

```diff
@@ -50,4 +56,28 @@ public long getColumnIndex(Class<? extends RealmModel> clazz, String fieldName)
             return -1;
         }
     }
+
+    public ColumnIndices copyDeeply() {
```

Comment:
I implemented that way at first, but `clone()` method of each class must call `super().clone()` in it and `super().clone()` has extra costs of some checks. So I stopped using `Cloneable`.